content/admin/onboard_team/CreateUsers.md

```diff
@@ -4,41 +4,108 @@ description: "Creating Users in Contrast TeamServer"
+### Individual users
 
-## Creating Users as an Organization Administrator
+To add a single user, click the button to **Add User** above the grid, and complete the following steps: 
 
-1. Log in to Contrast
-2. Navigate to the user menu in the upper right 
-3. Select **Organization Settings**
-4. Select **Users** in the left navigation
-5. Click the **Add User** button
-6. Provide the required inputs (Email, First & Last Name, and default Organization Role)
+* Enter the user's **First Name**, **Last Name** and **Email Address** in the provided fields. 
+* Check the box if you want to **Require email activation** instead of requiring a password.
+* Choose which of the **System Roles** should apply to the user in the dropdown menu. The default is **None**. 
+* Choose the **Organization** to which the user belongs. 
+* Once you decide on an organization, you can choose the default **Organization Role** in the dropdown menu as well as an **Application Access Group** in the multiselect field.
+* Choose the **Date Format**, **Time Format** and **Time Zone** in the dropdown fields. 
+* The box to **Use Organization Settings** is checked by default. You can uncheck the box to create your own settings using the **Access** toggle or the box to **make user API only**. 
+* Click the **Add** button to save the user's information. 
 
->**Note:** Make sure you verify the [Role](admin-manageorgsroleperm.html#roles) so that users will have the privileges to carry out the actions you intend.
+<a href="assets/images/User-system-admin.png" rel="lightbox" title="Add a user as a Super Administrator"><img class="thumbnail" src="assets/images/User-system-admin.png"/></a>
 
-<a href="assets/images/Create_User.png" rel="lightbox" title="Add User"><img class="thumbnail" src="assets/images/Create_User.png"/></a>
 
-Users will receive an activation email to join Contrast. You will be able to check their status from the main Users table to identify who is waiting activation, active/inactive, or is locked out of their account based on security policy.
+### Multiple users 
+
+To bulk add users, click the upload icon above the grid to import a spreadsheet with the users' information. The spreadsheet must be CSV format, and include the following fields for each user. **All field headings and values in the spreadsheet must be formatted as shown.** 
+
+* First Name 
+* Last Name 
+* Email 
+* Organization 
+* Organization Role <br> (**View**, **Edit**, **Rules_admin**, or **Admin**)
+
+> **Note:** The **Email** field appears as **Username** for users who have [HTTP Header](installation-setupauth.html#http-proxy) or [LDAP authentication](installation-setupauth.html) configured. 
+
+You may also include the following information for each user in the spreadsheet: 
```

Comment:
@kaitlynmortimer I would maybe be a little more explicit here. Something like "You may also choose to include the following optional information in the user spreadsheet. To do so, add a new column for each field you wish to define, taking care to name all headers and values precisely as shown. "

Reply:
@kaitlynmortimer I also wonder if we should add a subheader so that if users navigate to this article from the UI tooltip, they can easily scan and find the section on adding optional fields, with instructions on how to do so.